web/client/components/import/style/StylePanel.jsx

```diff
@@ -67,11 +67,19 @@ class StylePanel extends React.Component {
         useDefaultStyle: false,
         zoomOnShapefiles: true,
         overrideAnnotation: false,
-        initialLayers: []
+        initialLayers: [],
+        disableStyleCustomization: false
     };
 
     componentDidMount() {
         this.setState({initialLayers: [...this.props.layers]});
+        this.setState({currentActiveLayer: this.props.selected});
+        this.checkAndDisableStyleCustomization();
+    }
+    componentDidUpdate(prevProps) {
```

Comment:
i think there is no need for this

Reply:
@MV88 I need this trigger the `this.checkAndDisableStyleCustomization()` . How would you recommend I do it because without it. I cant get the check to run again

Reply:
Ohh I get it now. Just need to call `renderDisableStyleCustomization()` and it will always get the latest props.selected value. Thanks @MV88